src/ipelib/IPE_Plasma_Class.F90

```diff
@@ -1073,9 +1082,46 @@ SUBROUTINE Cross_Flux_Tube_Transport( plasma, grid, v_ExB, time_step, mpi_layer
           ENDIF
 
           IF( lp_min == 0 )THEN
+            write(6,*) 'GHGM Need to expand search ', mp , lp
 !           STOP 'lp-CFL > 1'
+            ! Check poleward
+            IF( theta_t0 <= colat_90km(lp-1) .AND. theta_t0 >= colat_90km(lp-2) )THEN
```

Comment:
`colat_90km(lp-1)` can cause an out-of-bound error and/or return corrupted data. The same can happen with `colat_90km(lp+2)` 
Please check if `lp > 1` and `lp < NLP-1`, or implement additional code to guard against both these conditions.